### PR TITLE
fix: Update KeyboardEventHandler

### PR DESCRIPTION
**Pull request by darcywong00**
Split out the two interfaces, and add `onPackageInstalled` and `onLexicalModelInstalled`

### CODE REVIEW

File: developer/engine/android/14.0/KeyboardEventHandler/onPackageInstalled.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ <dt><code><?php echo $param1 ?></code></dt>
+<dd>The information dictionaries of the keyboards installed from a keyboard package with keys and values defined as <code>List&lt;HashMap&lt;String key, String value&gt;&gt;</code>.</dd>
+</dl>
+
+<h2 id="Description" name="Description">Description</h2>
+<p>Implement this method to handle keyboard package installed event.</p>
+
+<h2 id="Examples">Examples</h2>
+
+<h3 id="Example:_Using_method” name="Example:_Using_method">Example: Using <code><?php echo $event.'()' ?></code></h3>
```
Comment (mcdurdin) [inline developer/engine/android/14.0/KeyboardEventHandler/onPackageInstalled.php:29]:
```suggestion
<h3 id="Example:_Using_method" name="Example:_Using_method">Example: Using <code><?php echo $event.'()' ?></code></h3>
```

**Review by mcdurdin — CHANGES_REQUESTED** [review]
Some changes please:

* Avoid using `.php` extension in links
* Consider writing new content in Markdown (use .md extension instead of .php)

**Comment by darcywong00** [comment]
I've converted the new content to .md. But I don't see how to set the `head(['title' => 'XYZ']);`

In the left navigation, both pages show as "Untitled page.md"

**Comment by mcdurdin** [comment]
> I've converted the new content to .md. But I don't see how to set the `head(['title' => 'XYZ']);`

Yeah, sorry about that. There's a standard .md metadata section you can add:

```
---
title: my super web page
---
```

For an example, see https://raw.githubusercontent.com/keymanapp/help.keyman.com/staging/developer/cloud/package-version/1.0/index.md

File: developer/engine/android/14.0/KeyboardEventHandler/onLexicalModelInstalled.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## Syntax
+```java
+onLexicalModelInstalled(List<HashMap<String key, String value>> lexicalModelsInstalled)
+```
+
+### Parameters
+`lexicalModelsInstalled`
+
+The information dictionaries of the lexical-models installed from a lexical-model package with keys and values defined as <code>List&lt;HashMap&lt;String key, String value&gt;&gt;</code>.</dd>
```
Comment (mcdurdin) [inline developer/engine/android/14.0/KeyboardEventHandler/onLexicalModelInstalled.md:16]:
```suggestion
The information dictionaries of the lexical-models installed from a lexical-model package with keys and values defined as `List<HashMap<String key, String value>>`.
```

**Review by mcdurdin — APPROVED** [review]
One line missed in the html->md changeover but LGTM

**Pull request merged**